### PR TITLE
Changed 'the the' to 'the'

### PR DESCRIPTION
**Pull request by RamchandraApte**
Changed 'the the' to 'the'

### CODE REVIEW

File: libs/libRocket/Include/Rocket/Core/Element.h [hunk 1]
```diff
@@ -524,7 +524,7 @@ class ROCKETCORE_API Element : public ScriptInterface
 	bool HasChildNodes() const;
 
 	/// Get a child element by its ID.
-	/// @param[in] id Id of the the child element
+	/// @param[in] id Id of the child element
```
Comment (DolceTriade) [inline libs/libRocket/Include/Rocket/Core/Element.h]: THe changes to librocket need to be made in the librocket repo or they will be lost next time I sync the files over.

Comment (RamchandraApte) [inline libs/libRocket/Include/Rocket/Core/Element.h]: Yes, I will do so.

**Comment by DolceTriade** [comment]
In this case, I don't think the multiple exclamation marks are typos.

**Comment by mbasaglia** [comment]
Closing this, none of the changes have an actual effect and many are irrelevant or incorrect anyway

**Pull request closed without merging**